src/components/Navbar.js

```diff
@@ -1,4 +1,5 @@
 import { NavLink, Outlet } from 'react-router-dom';
+import '../styles/Nav.css';
 
 const Navbar = () => (
```

Comment:
- [ ] The person icon seems to not be displayed on the browser. 


|  |   |
| ------------- | ------------- |
| Your Design | <img width="929" alt="image" src="https://github.com/Rifat8080/Bookstore-React-App/assets/52934128/6575ef0f-6207-409b-b2b6-b486227315e3">  |
| Zeplin Design  | <img width="807" alt="image" src="https://github.com/Rifat8080/Bookstore-React-App/assets/52934128/ad0945a7-a193-4b3c-86f1-e5f4e73a7f8c">  |